INSTALL.md

```diff
@@ -78,21 +81,35 @@ $ camlp5 -v
 Camlp5 version 7.03 (ocaml 4.02.3)
 ```
 
-Under Mac OS X, you may obtain ProofGeneral from http://proofgeneral.inf.ed.ac.uk/.
-It comes with installation instructions.  Your version of emacs determines
-which version of ProofGeneral you need, roughly, so some experimentation may be
-required; you may even need the current development version if your emacs is
-recent.
-
-
 ## Preparing for the installation under Linux
 
-Under Ubuntu or Debian, you may install ocaml (and ProofGeneral) with
+uUnder Ubuntu or Debian, you may install ocaml (and ProofGeneral) with
```

Comment:
"uUnder"?

Reply:
My Control key is no longer reliable, so more and more and find cruft like that is produced while I'm editing a file in emacs.